contracts/message-based/PriceAwareUpgradeable.sol

```diff
@@ -36,7 +36,19 @@ contract PriceAwareUpgradeable is OwnableUpgradeable {
     }
 
 
-    function getPriceFromMsg(bytes32 symbol) internal view returns (uint256) {
+    function getPricesFromMsg(bytes32[] memory symbols) internal view returns (uint32[] memory) {
+        return _getPricesFromMsg(symbols);
+    }
+
+
+    function getPriceFromMsg(bytes32 symbol) internal view returns (uint32) {
```

Comment:
We should check if we don't add too much gas while wrapping in array

Reply:
checked, difference ~2%